### PR TITLE
Perform project cleanup.

### PR DESCRIPTION
**Pull request by karelmaxa**
Following changes has been performed:

* Upgraded `wrensec-parent` upgraded to the latest release `3.3.0`
* Target Java version is taken from the parent (i.e. `8`)
* Introduced base GitHub Actions build pipeline with two JREs (`11` and `17`)
* Performed base cleanup of POM file
* Removed some legacy stuff

### CODE REVIEW

File: i18n-core/src/main/resources/javadoc/overview.html [hunk 1]
```diff
@@ -1,134 +0,0 @@
-<html>
```
Comment (pavelhoral) [inline i18n-core/src/main/resources/javadoc/overview.html:1]: This should be migrated to `package-info.java`. IMO this is CDDL content and not CCPL as this is not standard site content but JavaDoc content which is part of the codebase.

**Review by pavelhoral — CHANGES_REQUESTED** [review]

File: i18n-core/src/main/resources/javadoc/overview.html (hunk 1 above)
Comment (karelmaxa) [reply i18n-core/src/main/resources/javadoc/overview.html:1]: `package-info.java` files are already present in every module and contain relevant description (e.g. [i18n-core/package-info.java](https://github.com/WrenSecurity/wrensec-i18n-framework/blob/master/i18n-core/src/main/java/org/forgerock/i18n/package-info.java)). 
Generation of sites have been fixed and the result looks fine.

Comment (karelmaxa) [reply i18n-core/src/main/resources/javadoc/overview.html:1]: Javadoc overview file for `i18n-core` was reverted back and moved to the correct location.

**Review by pavelhoral — APPROVED** [review]
LGTM

**Pull request merged**